test/Numerics/Mesh/Geometry.jl

```diff
@@ -45,15 +45,16 @@ MPI.Initialized() || MPI.Init()
         (zmax - zmin) / (polynomialorder * Ne[3]),
     )
     Savg = cbrt(prod(S))
+    Shoriavg = (S[1] + S[2]) / 2
     M = SDiagonal(S .^ -2)
 
-    N = polynomialorder
-    Np = (N + 1)^3
+    N = (polynomialorder, polynomialorder)
+    Np = (polynomialorder + 1)^3
     for e in 1:size(grid.vgeo, 3)
         for n in 1:size(grid.vgeo, 1)
             g = LocalGeometry{Np, N}(grid.vgeo, n, e)
             @test lengthscale(g) ≈ Savg
-            @test Geometry.resolutionmetric(g) ≈ M
+            @test lengthscale_horizontal(g) ≈ Shoriavg
```

Comment:
Why is the geometric average (via resolutionmetric)check being removed from the test here?

Reply:
Because it's not updated for the horizontal and vertical splitting. I checked with @jkozdon but currently we don't know how to change it. On the other hand, `resolutionmetric` is not used anywhere in the code.